docs/examples/kotest/src/test/kotlin/com/example/kotest/TestContainerKotestIntegrationPerTest.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.kotest
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.delay
+import org.testcontainers.containers.ContainerPerTestListener
+import org.testcontainers.containers.GenericContainer
+
+class TestContainerKotestIntegrationPerTest : StringSpec({
+    val container = GenericContainer<Nothing>("ubuntu")
+    setContainerCommand(container) // Set container command for first time to echo hello Kotest
+
+    // Register the Listener
+    listeners(ContainerPerTestListener(container))
```

Comment:
I believe more idiomatic way would be to have something like this:
```kotlin
listeners(container.perTest())
```

where:
```kotlin
fun Startable.perTest() = ContainerPerTestListener(this)
```

Reply:
Done